### DISCUSSION TITLE
I am not able to verify my 2 factor authentication in github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#101942] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Feature Area

Issues

### Body

Hey this is Prasiddha From Nepal
I am trying to set up my 2-way verification for GitHub and every time I try to confirm the setup key the button is disabled and I am not able to continue further process. It would be a great help if u could resolve it. 
![image](https://github.com/community/community/assets/53443661/bb7d8978-8eb6-4f5c-a7af-c10de2887aec)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by amankr1469** [comment]
Ensure there are no typos or extra spaces in the key Or Try using a different browser to access GitHub.

**Comment by ghost** [comment]
I have don both but nothing is working for me

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Prasiddha777 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by ghost** [comment]
Thanks a lot.
